src/test/java/net/snowflake/client/jdbc/SnowflakeResultSetSerializableIT.java

```diff
@@ -625,7 +630,10 @@ public void testCloseUnconsumedResultSet() throws Throwable {
   @Test
   @ConditionalIgnoreRule.ConditionalIgnore(condition = RunningOnGithubAction.class)
   public void testNegativeWithChunkFileNotExist() throws Throwable {
-    try (Connection connection = init()) {
+    // This test takes about (download worker retry times * networkTimeout) long to finish
+    Properties properties = new Properties();
+    properties.put("networkTimeout", 10000); // 10000 millisec
```

Comment:
So this failed because we had no network timeout set?

Reply:
Yeah, without the fix, the chunk download workers just do 10 retries that all end up with 403 very quickly. But after the fix, since we now would retry 403, it takes a chunk download worker 1 hour to throw an error after receiving nothing on the first try.